TestApplication/fa.lproj/MainMenu.strings

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
this file looks untranslated

Reply:
They're all untranslated for the TestApplication MainMenu.strings. This is needed in order for the other Sparkle localization strings to be activated in the test app, so works as intended.